src/core/transforms/Overlay.tsx

```diff
@@ -66,6 +66,7 @@ export const Overlay = {
         <React.Fragment>
           {meta?.type === 'html-overlay' && (
             <Iframe
+              allow='autoplay'
```

Comment:
My one concern is that In the past we have had issues with browsers not allowing `autoplay`.

Reply:
@mikekanter Understand your concerns, but this is tested at-least on chrome and it's working. The same behaviour is implemented in studio1 as well.